### DISCUSSION TITLE
I recorded to a click track and it immediately fell out of sync on playback

### DISCUSSION THREAD

**Discussion by sdfalk** [helgoboss/helgobox#1138] [General]
I guess the title pretty much sums up my issue. 
I played to a click track ,and after I start and stop again, It fall out of sync

**Comment by helgoboss** [comment]
What exactly did you do? A screen recording along with some explanation where to look at would be helpful.

**Reply by sdfalk** [reply]
Please see attachment..Im sure Im doing something wrong, Im just not sure what

https://github.com/user-attachments/assets/717669d9-cd1b-40b7-8662-c8c6e1e00cdd

**Reply by sdfalk** [reply]
So I loaded up a virtual instrument (Klevgrand ons shot) and just began recording a quick 2 bar loop.  I recorded 2 bars, stopped the recording , played it back, and as you can hear it immediately falls out of sync  with the metronome.
Im still so new to your app, Im not even sure how to change the length of the loop within the app (or if thats possible) so there's a possibility I'm missing something.

**Reply by helgoboss** [reply]
You are recording while Playtime is stopped. Currently, this triggers a special feature, the so-called **tempo detection recording** (did you see the red message in the title bar?). With such a tempo detection recording, you need to be extremely careful when to press record and when to press stop, because it will dictate your new project tempo.

In addition to that, you are clicking on the tempo tap button while the recording is already ongoing. Tempo changes during recording destroy the timing for sure!

To sum it up, "Tempo detection recording" is not what you want! In order to get perfectly timed recordings,  you currently need to start recording while Playtime is already playing and its metronome is on. Please check out my Playtime 2 video (the video on the website), you see the process there. It's showed in the first few minutes.

The next update of Playtime adds the possibility to do recordings to a click **from stopped state**. I think you will like that.

**Reply by sdfalk** [reply]
Sounds good..thanks for the reply. Im trying to keep my workflow within one daw, so I'm less in need of Ableton Live.
